browser.js

```diff
@@ -0,0 +1,35 @@
+/* globals document, ImageData */
+
+const parseFont = require('./lib/parse-font')
+
+exports.parseFont = parseFont
+
+exports.createCanvas = function (width, height) {
+  return Object.assign(document.createElement('canvas'), { width, height })
+}
+
+exports.createImageData = function (array, width, height) {
+  // Browser implementation of ImageData looks at the number of arguments passed
+  switch (arguments.length) {
+    case 0: return new ImageData()
+    case 1: return new ImageData(array)
+    case 2: return new ImageData(array, width)
+    default: return new ImageData(array, width, height)
```

Comment:
`0` and `1` doesn't exist, I added them to get the errors reported back to be accurate.

![screen shot 2017-06-24 at 21 31 52](https://user-images.githubusercontent.com/189580/27511444-91e83b98-5924-11e7-9e53-1c3f8f9d879f.png)

`return new ImageData(...arguments)` would be perfect, but unfortunately Node.js 4.x doesn't support it... It's doing this since passing `(10, 10, undefined)` doesn't do what you want it to in the browsers:

![screen shot 2017-06-24 at 21 33 28](https://user-images.githubusercontent.com/189580/27511451-cab0d9e4-5924-11e7-9e73-d71b9cfa11c1.png)

Not 100% sure that this is the best approach though 🤔